### PR TITLE
create live chart and get data to visualize in chart

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

File: src/pages/SwapV2/index.tsx [hunk 1]
```diff
@@ -79,17 +74,20 @@ enum ACTIVE_TAB {
...
+  @media only screen and (min-width: 768px) {
+    width: 404px;
+  }
 `
 
 export default function Swap({ history }: RouteComponentProps) {
   const [rotate, setRotate] = useState(false)
   const [showInverted, setShowInverted] = useState<boolean>(false)
-  const [showRoute, setShowRoute] = useState<boolean>(false)
+  const [isOpenChart, setIsOpenChart] = useLocalStorageState<boolean>('isOpenChart', isMobile ? false : true)
```
Comment (viet-nv) [inline src/pages/SwapV2/index.tsx:87]: I think we should move this to user state on Redux for consistent

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -135,10 +135,12 @@
     "@sushiswap/sdk": "^5.0.0-canary.6",
     "@types/big.js": "^6.0.0",
     "@types/react-copy-to-clipboard": "^5.0.2",
+    "@types/recharts": "^1.8.23",
     "@uniswap/default-token-list": "^2.0.0",
     "@uniswap/sdk": "^3.0.3",
     "dayjs": "^1.10.7",
     "env-cmd": "^10.1.0",
+    "framer-motion": "4.1.17",
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply package.json:143]: Current our bundle size is too large 3.7MB, That why I dont want to add new library while we already have an other lib with same purpose 
image

File: src/components/LiveChart/index.tsx [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+                alignItems={'center'}
+                justifyContent={'center'}
+                color={theme.disableText}
+                style={{ gap: '16px' }}
+              >
+                {loading && <Loader />}
+                {error && (
+                  <>
+                    <WarningIcon />
+                    <Text fontSize={16}>There was an error with the chart. Try again later</Text>
```
Comment (viet-nv) [inline src/components/LiveChart/index.tsx:229]: Please wrap an `<Trans></Trans>` tag for multi language functionality

Comment (XiaoYhun) [reply src/components/LiveChart/index.tsx]: I fixed all the feedbacks, please recheck, thanks

File: .env [hunk 4]
```diff
@@ -4,7 +4,7 @@ REACT_APP_NETWORK_URL="https://dev-ethereum.knstats.com/v1/mainnet/geth?appId=de
 REACT_APP_MIGRATOR_ADDRESS=0x247B641bB4eAff621987E2B5c3D0247489556E75
 REACT_APP_MAINNET_ENV=staging
 REACT_APP_INFURA_KEY="9aa3d95b3bc440fa88ea12eaa4456161"
-REACT_APP_AGGREGATOR_API="https://dev-dmm-aggregator.knstats.com"
+REACT_APP_AGGREGATOR_API="https://aggregator-api.kyber.org"
```
Comment (piavgh) [inline .env:7]: we should not change this. It's for dev environment

File: .env.ropsten [hunk 5]
```diff
@@ -6,7 +6,7 @@ REACT_APP_FORTMATIC_KEY="pk_live_EF523DF5F8AC3058"
 REACT_APP_MIGRATOR_ADDRESS=0x247B641bB4eAff621987E2B5c3D0247489556E75
 REACT_APP_MAINNET_ENV=staging
 REACT_APP_INFURA_KEY="9aa3d95b3bc440fa88ea12eaa4456161"
-REACT_APP_AGGREGATOR_API="https://dev-dmm-aggregator.knstats.com"
+REACT_APP_AGGREGATOR_API="https://aggregator-api.kyber.org"
```
Comment (piavgh) [inline .env.ropsten:9]: same here, only .env.production should use the production API

File: src/constants/index.ts [hunk 6]
```diff
@@ -932,7 +942,8 @@ export const COINGECKO_NETWORK_ID = {
   [ChainId.FANTOM]: 'fantom',
   [ChainId.CRONOS]: 'cronos',
 
-  [ChainId.ROPSTEN]: '',
+  //set mainnet for testing purpose
+  [ChainId.ROPSTEN]: 'ethereum',
```
Comment (piavgh) [inline src/constants/index.ts:946]: when this PR is ready to be merged, we should change this back

File: src/state/application/reducer.ts [hunk 7]
```diff
@@ -93,4 +104,12 @@ export default createReducer(initialState, builder =>
     .addCase(setGasPrice, (state, { payload: gasPrice }) => {
       state.gasPrice = gasPrice as GasPrice
     })
+    .addCase(toggleLiveChart, (state, action) => {
```
Comment (viet-nv) [inline src/state/application/reducer.ts:107]: I think this should belong to user's state, not application's state. 
https://github.com/dynamic-amm/dmm-interface/blob/main/src/state/user/reducer.ts#L63..L75
And currently, we are persisting user setting, so if you move this to user's state, you do not need to set it manually. 
https://github.com/dynamic-amm/dmm-interface/blob/main/src/state/index.ts#L18

Comment (viet-nv) [reply src/state/application/reducer.ts:107]: This is the last thing I wanna change @XiaoYhun, These other things look good to me :D

Comment (XiaoYhun) [reply src/state/application/reducer.ts:107]: okay didn't recognize it, thanks

File: .env (hunk 4 above)
Comment (viet-nv) [reply .env:7]: for local development, If you want to run with production ENV, you can run the cmd yarn start-production @XiaoYhun :D

**Review by viet-nv — APPROVED** [review]

**Pull request merged**